components/automate-chef-io/content/docs/applications-dashboard.md

```diff
@@ -95,7 +95,7 @@ Applying the filter changes the Service Groups list to display matching results
 
 Disconnected services are defined services that have not reported back with a health check message for a user-defined period of time.
 
-Use the Chef Automate Service Groups API endpoint to manage the activity of periodic jobs, and to assess disconnection status and removal of services.
+Use the [Service Group Data Lifecycle settings](https://automate.chef.io/docs/data-lifecycle/#service-groups) to manage the activity of periodic jobs, and to assess disconnection status and removal of services.
```

Comment:
Question to y'all - With this recommendation of using the UI to manage settings in line 98, do we want to remove references of API management elsewhere in this page? See line 122 (Marking a Service as Disconnected) and line 126 (Periodic Removal)

Reply:
Content after line 122 is now removed! Thanks @danielsdeleo!